app/src/main/res/layout/share_tab_item.xml

```diff
@@ -42,7 +42,7 @@
         android:textSize="12sp"
         android:maxLines="1"
         android:ellipsize="end"
-        android:textColor="?secondaryText"
+        android:textColor="@color/photonLightGrey60"
```

Comment:
Does this work for both light and dark theme? Changes like these are very hard because we have to consider the light and dark theme. The other consideration is the secondaryText color. It would be interesting to audit all the UI elements that use the secondaryText color and see if we run into similar text contrast issues. However, if this only affects the current UI, then the changes should be fine.

Reply:
@gabrielluong Yes, this change of colors has fixed corresponding accessibility issues in both light and dark theme. For the attached screenshots, accessibility issues arising due to contrast issues were no more being shown by Google Accessibility Scanner 👍 
Shall I try to find out if there are other occurrences of text contrast issues due to `secondaryText` color as a part of this PR/issue itself?

Reply:
No, this is good to fix this issue. If we find any other contrast issues we can file a new bug.

Reply:
Okay 👍